### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by maxim-lobanov**

### CODE REVIEW

**Review by AlenaSviridenko — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @actions/virtual-environments-owners
```
Comment (gtothe7) [inline .github/CODEOWNERS:1]: Change made

**Review by gtothe7 — CHANGES_REQUESTED** [review]
Change made

File: .github/CODEOWNERS (hunk 1 above)
Comment (gtothe7) [inline .github/CODEOWNERS:1]: Code changes to multiple lines

Comment (gtothe7) [inline .github/CODEOWNERS:1]: - 

>

**Review by gtothe7 — COMMENTED** [review]
> 

>

**Review by gtothe7 — APPROVED** [review]

**Review by gtothe7 — APPROVED** [review]
[<>]()

**Review by gtothe7 — COMMENTED** [review]
Thanks

File: .github/CODEOWNERS (hunk 1 above)
Comment (ubaistp) [reply .github/CODEOWNERS:1]: Uba

**Pull request merged**